src/jbi/bugzilla.py

```diff
@@ -91,6 +91,7 @@ class BugzillaBug(BaseModel):
     whiteboard: Optional[str]
     keywords: Optional[List]
     flags: Optional[List]
+    groups: Optional[List]
```

Comment:
nitpick: we could leave for another pull-request if we don't really use them. Also, it's not harmful.

Reply:
True, do you want me to remove it?

Reply:
You can leave it, it just affects parsing